launch/rslidar_bringup.launch.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2024 Trossen Robotics - All Rights Reserved
+#
+# Unauthorized copying of this file, via any medium is strictly prohibited
+# Proprietary and confidential
+
```

Comment:
Remove all of our proprietary license declarations.

```suggestion
```

Reply:
Just remove? I'm not sure if we can put their license on this. Should I also remove them from our lifecycle node files?

Reply:
You should remove our proprietary license declarations everywhere since this will be open source. If not specified, all files will be under the repo license. We could add the original license but add ourselves to it if the file was changed significantly.